test/testinput/bufr_bufr_backend_sfcsno.yaml

```diff
@@ -0,0 +1,20 @@
+time window:
+  begin: "2018-04-14T21:00:00Z"
+  end: "2023-12-15T03:00:00Z"
```

Comment:
I'll suggest adding:
`  bound to include: begin
`

Reply:
Will do